WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -503,7 +503,7 @@ function WeakAuras.IsOptionsOpen()
   end
 end
 
-function WeakAuras.GetSortedOptionsLists()
+local function GetSortedOptionsLists()
```

Comment:
Now this _will_ break at least one aura I know of. But that aura was manipulating WeakAuras to convince the user that it wasn't installed, so I'm ok with breaking that.

Reply:
That's a good idea then :)